### PR TITLE
Newsletter 7: Add Veloren February content

### PR DESCRIPTION
**Pull request by AngelOnFira**
Part of #64 ("Newsletter 7: Coordination/Tracking")

This includes a Youtube iframe template. Hopefully it's ok to add.

### CODE REVIEW

**Comment by ozkriff** [comment]
> This includes a Youtube iframe template. Hopefully it's ok to add.

Hmm. I've been deliberately trying to avoid everything that GitHub's Markdown render can't handle - so previews & diffs stay useful.

@17cupsofcoffee @Lokathor @AlexEne @erlend-sh - what do you think?

File: content/posts/newsletter-007/index.md [hunk 1]
```diff
@@ -169,6 +169,63 @@ February's full devlogs:
...
+- [Settlements](https://veloren.net/devblog-54#settlements)
+- [Mac Builds](https://veloren.net/devblog-55#the-mac-build-situation-by-angelonfira)
+- [Achievements](https://veloren.net/devblog-55#achievement-implementation-by-thedip)
+- [Persistence](https://veloren.net/devblog-56#persistence-by-angelonfira)
+
+![Hanging out](veloren2.png)
+
+In March, there will be a heavy focus on completing level of detail work as well as persistence. Modding support will be explored further. Player achievements will be merged into the game, and we will start looking at where more content can be added.
+
+January's full weekly devlogs: "This Week In Veloren...":
```
Comment (ozkriff) [inline content/posts/newsletter-007/index.md]:
```suggestion
February full weekly devlogs: "This Week In Veloren...":
```

Comment (AngelOnFira) [reply content/posts/newsletter-007/index.md]: Thanks :P

**Comment by AlexEne** [comment]
I would go for the ease of preview, you can leave it as a link

**Comment by 17cupsofcoffee** [comment]
I have no philosophical objection to including YouTube embeds in the newsletter, but would also slightly lean towards doing them in a way that shows up on GitHub's PR preview if at all possible.

If you put the embed HTML directly in the markdown file, does it show up on GitHub? If so, maybe we could keep the CSS classes and provide an example of how to use them in the 'ideal section structure' at the top of the newsletter template? Might be less maintainable though, somewhat torn 😅

**Comment by ozkriff** [comment]
> If you put the embed HTML directly in the markdown file, does it show up on GitHub?

Nope, GitHub's Markdown doesn't support it.

**Comment by 17cupsofcoffee** [comment]
Bah, understandable (would probably be a security nightmare) but annoying!

**Comment by AngelOnFira** [comment]
I understand that it would be great to be able to preview what it looks like in the PR, but it really seems like we'd be shooting ourselves in the foot on this one. Youtube videos are a good standard of something that plays videos on the newsletter, and I imagine there will be times that this comes up again in the future.

As far as I can find, there isn't anything that can embed youtube videos into Github markdown without doing an external link.

**Comment by AlexEne** [comment]
There are a bunch of other youtube vids linked with normal links from the newsletter, so I think it's fine to leave this one with a link too (just like the others). 

As a bonus it also makes for easier diffs like @ozkriff  said.

**Comment by ozkriff** [comment]
I usually used either [GIFs](https://github.com/rust-gamedev/rust-gamedev.github.io/blob/1432f478f80950e798e114d13db59356982508ed/content/posts/newsletter-001/index.md#bonus) or [youtube screenshots](https://github.com/rust-gamedev/rust-gamedev.github.io/blob/1432f478f80950e798e114d13db59356982508ed/content/posts/newsletter-007/index.md#rusty-shooter) for linking to videos.

**Comment by 17cupsofcoffee** [comment]
Another potential consideration would be that if we allowed YouTube embeds, we'd have to strip them out if we ever started [sending the newsletter out as an email](https://github.com/rust-gamedev/rust-gamedev.github.io/issues/24), as [most email clients block iframes](https://mailchimp.com/help/limitations-of-html-email/). That kinda makes me lean towards @ozkriff's suggestion, although I agree it's not ideal to not have the videos inline.

**Comment by AngelOnFira** [comment]
kk sounds good, I'll go for the gif option

**Comment by AngelOnFira** [comment]
@ozkriff Should be good to go now :)

**Pull request merged**